pkg/sql/logictest/testdata/logic_test/alter_table

```diff
@@ -1763,3 +1763,13 @@ ALTER TABLE t60786 DROP CONSTRAINT uq CASCADE
 
 statement ok
 ROLLBACK
+
+# Visible values can be added and removed after table creation.
+statement ok
+CREATE TABLE visible_table (a int primary key)
+
+statement ok
+ALTER TABLE visible_table ALTER COLUMN a SET VISIBLE
```

Comment:
can you add a

```
query TT
SHOW CREATE TABLE
----
```

after both ALTER statements, and then run `make testccllogic FILES='alter_table_locality'  TESTFLAGS='-rewrite'` to make sure they match expectations?

Reply:
can you also add tests that SET VISIBLE/NOT VISIBLE on already VISIBLE/NOT VISIBLE columns respectively (i.e. check they are idempotent)